scss/mixins/_hover.scss

```diff
@@ -8,16 +8,15 @@
 //    }
 //  }
 //  @else {
-    &:hover { @content }
+  &:hover { @content }
```

Comment:
This change should be removed; it's purposefully aligned with the commented out parts above and below it.